### ISSUE TITLE
FM Towns  emulation in emudeck

### ISSUE THREAD

**Issue by 13GingeR** [dragoonDorise/EmuDeck#667]
I’d love to see emudeck native support for FM Towns - core/emulator
we have fmtowns folder in roms along other platforms

i’d love to play someday Dragons Of Flame cd version on emudeck. i own Amiga version but fmtowns looks far superior comparing to 16 bit amiga or pc, not mentioning 8bit versions

**Comment by WingofaGriffin** [comment]
This will need to be tested with the binary provided by the suggested emulator: https://github.com/captainys/TOWNSEMU/releases

**Comment by 13GingeR** [comment]
it works like a charm on steam deck: i used Tsugaru windows binary with game iso and it’s working. if there is a way to get it working through emudeck it would be great!

**Comment by WingofaGriffin** [comment]
I'm going to need help with this: do you have a config file handy for best settings for deck and setting up controls? I've started a draft of getting this set up here: https://github.com/dragoonDorise/EmuDeck/pull/704/files but I am completely unfamiliar with this emulator and system

**Comment by 13GingeR** [comment]
![F1D0E8E7-DA38-47DD-9D8D-BB381E5FAED3](https://user-images.githubusercontent.com/4238489/235419618-1e1bdf37-ea7e-48f8-9a90-6cb83289a0f1.jpeg)

![BFA012B9-1F49-49CB-AD3C-099DE2640221](https://user-images.githubusercontent.com/4238489/235419788-58b2df34-00da-4389-885e-17777faa56f1.jpeg)
![51CEEBA7-3AD2-40E8-A4F4-12007BFFB931](https://user-images.githubusercontent.com/4238489/235419795-bef65a65-b045-4062-a641-7dabdf1c6e5f.jpeg)

these are my settings that let me play on steamdeck: bios(rom) folder, FD0 for saving (this disk is in tsugaru emu archive) - cd image with game..

game port as „gamepad by host game cotroller 0” and it works with steamdeck controls

**Comment by 13GingeR** [comment]
i cant see any cfg file inside emu folder but somehow my settings are stored.

there is one more option to set - autoload/autostart. i’m not using it at the moment becouse i had some tweaking fun with settings

![6BA80267-8B1D-4C21-8DC6-23204A3FE277](https://user-images.githubusercontent.com/4238489/235420281-308fd0de-28e3-4f82-bf50-c170fa2ec27e.jpeg)

**Comment by Ardend** [comment]
Will it be possible to run the windows version through EmulationStation via Emudeck?  EmulationStation on the deck seems to only look for the Linux apps, but maybe there is a config to change to have it look for the .exe versions?

**Comment by WingofaGriffin** [comment]
Yes, it will be possible t run via EmulationStation. For Proton ran emulators, we create a custom entry that lets you play them.

**Comment by Ardend** [comment]
Perfect, looking forward to it.  The Linux control issues don't seem like they will ever get fixed.  Oddly enough, the touch screen interacts with the games in the Linux version, but its random on what it does depending on the game. Controllers don't work at all still though.
